modules/import/csv_file.py

```diff
@@ -31,8 +32,8 @@ def __init__(self, *args, **kwargs):
         self.__init_options()
         return result
 
-    def do_set(self, *args, **kwargs):
-        BaseModule.do_set(self, *args, **kwargs)
+    def _do_options_set(self, *args, **kwargs):
```

Comment:
Rename to match https://github.com/lanmaster53/recon-ng/blob/master/recon/core/framework.py#L889